devicetypes/juano2310/jawbone-user.src/jawbone-user.groovy

```diff
@@ -4,7 +4,7 @@
  *  Author: juano23@gmail.com
  *  Date: 2013-08-15
  */
- // for the UI
+ //fortheUI
```

Comment:
?

Reply:
I had bug somewhere in script that for some reason removes all spaces from couple of comments (afaik total 3 in whole repository, dunno why)...